controllers/taskController.js

```diff
@@ -33,28 +17,26 @@ const getByID = (req, res) => {
 };
 
 const create = (req, res) => {
-    const title = req.body.title;
-    const course = req.body.course;
-    const deadline = req.body.deadline;
-    let task = {
-        id: tasks.length+1,
-        title: title,
-        course: course,
-        deadline: deadline
-    }
-    tasks.push(task);
+    let props = {
```

Comment:
this should be a `const` cause it doesn't change. You could just do it inline in the taskRepo call below though since the scope is so small.